documentation/environment-operations-center-guide/secure-data-connectors/deploy-sdc-client.md

```diff
@@ -80,18 +80,177 @@ To deploy the secure data connector client on a Linux system, first unzip the *s
+
+![image description](images/linux-dotnet-run-client.png)
 
 Once the client is running, you can can setup a connection with the on-premise backend. For details on setting up a connection, see the [server backend](../../sys-admin-guide/server-backend.md) guide.
 
+### Deploy as a daemon on Linux
+
+#### Step 1: CD to binary location and give execute permissions for inlets-pro and RadiantLogic.OnPremisesAgentClient.Agent
+
+`chmod +x ./RadiantLogic.OnPremisesAgentClient.Agent`
+
+`chmod +x ./inlets-pro`
+
+![image description](images/linux-chmod-cmds.png)
+#### Step 2: Check the Init System:
+Determine which init system your Linux distribution uses. Common init systems include systemd, SysV init, and Upstart. You can often identify the init system by checking the version of Linux or the documentation for your specific distribution. Steps 3.1 and 3.2 covers the cases of systemd and SysV Init.
+
+#### Step 3.1: If your system use 'systemd'. Follow below steps:
+
+**Create a Systemd Service File:**
+
+Create a .service file in the /etc/systemd/system/ directory. This file will define the configuration for our application daemon. Replace **‘your-app-name’** with an appropriate name for SDC Client:
+
+`sudo nano /etc/systemd/system/your-app-name.service`
+
+**Edit the Service File:**
+
+In the nano editor, add the following content to the service file, adjusting the paths and parameters to match your environment:
+
+```
+[Unit]
+Description=Give your description here for running SDC Client as Daemon
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+WorkingDirectory=/path/to/binary/sdc-client
+ExecStart=/usr/bin/dotnet /path/to/binary/sdc-client/RadiantLogic.OnPremisesAgentClient.Agent.dll
+Restart=always
+RestartSec=10
+SyslogIdentifier=your-app-name
+User=your-username
+Environment=ASPNETCORE_ENVIRONMENT=Production
+
+[Install]
+WantedBy=multi-user.target
+```
+
+![image description](images/linux-edit-service-file.png)
+
+- WorkingDirectory: Set this to the directory where your downloaded binaries is located.
+
+- ExecStart: Specify the path to executable and the path to application's DLL.
+
+- User: Replace your-username with the appropriate username that should run the application.
+
+**Reload Systemd and Start the Service:**
+
+After creating and editing the service file, reload the systemd configuration and start the application: 
+
+`sudo systemctl daemon-reload`
+
+`sudo systemctl start your-app-name`
+
+![image description](images/linux-start-daemon.png)
+
+**Enable Autostart on Boot:**
+
+To make sure your application starts automatically when the system boots, enable the service:
+
+`sudo systemctl enable your-app-name`
+
+To validate the sdc client is successfully started as a service, you can check the client logs in the /Logs folder and opening the log file:
+
+![image description](images/linux-check-logs-service-started.png)
```

Comment:
Minor: If possible when you have the portforwarder added to capture inlets-pro running for standalone linux client, can we have the logs captured with inlets running for daemon as well?